app/modules/elocaps/cruds_elocaps.py

```diff
@@ -0,0 +1,271 @@
+from collections.abc import Sequence
+from datetime import date
+
+from sqlalchemy import desc, func, not_, select
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import selectinload
+
+from app.modules.elocaps import models_elocaps, schemas_elocaps
+from app.modules.elocaps.types_elocaps import CapsMode
+
+
+async def get_latest_games(db: AsyncSession, count=10) -> Sequence[models_elocaps.Game]:
+    result = await db.execute(
+        select(models_elocaps.Game)
+        .order_by(desc(models_elocaps.Game.timestamp))
+        .options(
+            selectinload(models_elocaps.Game.game_players)
+            .selectinload(models_elocaps.GamePlayer.player)
+            .selectinload(models_elocaps.Player.user),
+        )
+        .limit(count),
+    )
+    return result.scalars().all()
+
+
+async def get_games_played_on(
+    db: AsyncSession,
+    time: date,
+) -> Sequence[models_elocaps.Game]:
+    result = await db.execute(
+        select(models_elocaps.Game)
+        .where(func.DATE(models_elocaps.Game.timestamp) == time)
+        .options(
+            selectinload(models_elocaps.Game.game_players)
+            .selectinload(models_elocaps.GamePlayer.player)
+            .selectinload(models_elocaps.Player.user),
+        ),
+    )
+    return result.scalars().all()
+
+
+async def get_player_info(
```

Comment:
Could you add a docstring explaining should be returned by this function. I think the idea is to return a Player object per game mode